fmings/limeify-be
# Is public toggle fm

## Description
Added key to Playlist model for IsPublic
Added key to UpdatePlaylist method to allow users to toggle if the playlist is public or private
Added new API call to get all Public playlists

## Related Issue
#34 

## Motivation and Context
This allows users to toggle playlists they created to be public or private.
It also allows users to view any public playlists.

## How Can This Be Tested?
Run program
Test "update playlist" api in swagger to confirm you can toggle IsPublic between true and false
Test "get public playlists" api in swagger to confirm public playlists are retrieved

## Screenshots (if appropriate):

## Types of changes
